### PR TITLE
Added toggle button to switch between login and signup page

### PR DESCRIPTION
**Pull request by Mastermind-sap**

### CODE REVIEW

File: lib/pages/login/login_page.dart [hunk 1]
```diff
@@ -45,7 +47,18 @@ class LoginPage extends StatelessWidget {
...
-            const GoogleSignInButton()
+            Column(
+              children: [
+                const GoogleSignInButton(),
+                // Toggle button to signup page
+                TextButton(
+                  onPressed: () {
+                    Navigator.pushNamed(context, SignUpPage.routeName);
+                  },
+                  child: const Text("Don't have an account? Sign Up"),
```
Comment (Raj217) [inline lib/pages/login/login_page.dart]: Underline Sign Up
If you want you can use RichText

File: lib/pages/signup/signup_page.dart [hunk 2]
```diff
@@ -111,6 +112,14 @@ class _SignUpPageState extends State<SignUpPage> {
                           child: const Text("Sign Up"),
                         ),
 
+                        // Toggle button to login page
+                        TextButton(
+                          onPressed: () {
+                            Navigator.pushNamed(context, LoginPage.routeName);
```
Comment (Raj217) [inline lib/pages/signup/signup_page.dart]: Dont use pushNamed
User can go back to the sign up page on back 
Fine for 1 or 2 times but maybe pushReplacement would be better

**Review by Raj217 — CHANGES_REQUESTED** [review]

File: lib/pages/signup/signup_page.dart (hunk 2 above)
Comment (Mastermind-sap) [reply lib/pages/signup/signup_page.dart]: Fixed [here](https://github.com/gdsc-nits-org/efficacy_admin/pull/20/commits/8b92d53731a18c935a0100e45cfb98f210c629b8)

File: lib/pages/login/login_page.dart (hunk 1 above)
Comment (Mastermind-sap) [reply lib/pages/login/login_page.dart]: Fixed [here](https://github.com/gdsc-nits-org/efficacy_admin/pull/20/commits/e454175105909e82cbb853b1e0f02321f332daad)

**Pull request merged**